File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int num = get_int();

    while num > 0
    {
        printf(" " * (num - 1) + "#" + "  " + " " * (num - 1)  + "\n");
        num --;
    }
}

File: Week 6 - Python/sentimental-mario-more/mario.py
from cs50 import get_int

h = 0
while h < 1 or h > 8:
    h = get_int("Height: ")

for i in range(h):
    blank = " " * (h - 1 - i)
    s = "#" * (i + 1)
    print(f"{blank}{s}  {s}")
